ripe/atlas/tools/commands/probes.py

```diff
@@ -142,17 +146,16 @@ def run(self):
         self.renderer.on_start()
 
         if self.arguments.aggregate_by:
-
             aggregators = self.get_aggregators()
             buckets = aggregate(probes, aggregators)
             self.renderer.render_aggregation(buckets)
         else:
-
             self.renderer.on_table_title()
```

Comment:
I like the new line, but then I would put an additional new line above line 152 as well to keep things symetrical.  I can undo this bit though if it makes you sad.